cloudsim-plus-testbeds/src/main/resources/instances-files.txt

```diff
@@ -0,0 +1,7 @@
+t2.nano.json
```

Comment:
Rename the file to "instance-files.txt".
Remember to change the name where you are using the file.